includes/apple-exporter/class-exporter-content.php

```diff
@@ -117,14 +125,16 @@ public static function format_src_url( $url ) {
 	 * @param int                      $id       The ID of the post to be exported.
 	 * @param string                   $title    The title of the post to be exported.
 	 * @param string                   $content  The content of the post to be exported.
+	 * @param string                   $slug     Optional. The slug of the post to be exported.
 	 * @param string                   $intro    Optional. The intro of the post to be exported.
 	 * @param string|array             $cover    Optional. The cover of the post to be exported. If string, just the URL. If array, properties are 'url' and 'caption'.
 	 * @param string                   $byline   Optional. The byline of the post to be exported.
 	 * @param \Apple_Exporter\Settings $settings Optional. Settings for the exporter.
```

Comment:
Reorder parameters in docblock:
```suggestion
	 * @param string                   $intro    Optional. The intro of the post to be exported.
	 * @param string|array             $cover    Optional. The cover of the post to be exported. If string, just the URL. If array, properties are 'url' and 'caption'.
	 * @param string                   $byline   Optional. The byline of the post to be exported.
	 * @param \Apple_Exporter\Settings $settings Optional. Settings for the exporter.
	 * @param string                   $slug     Optional. The slug of the post to be exported.
```